Using worf is so much better

CestDiego opened the pull request:
[worf-mode](http://oremacs.com/worf/README.html) gives more vi-like
key bindings for org-mode :D evil-org on the other hand shadows some
long stablished vim keybindings :) I feel you should try it. there is
not really a conf for it so maybe just try it?

THIS DELETES EVIL-ORG

syl20bnr commented:
I will put a break on this and take time to think about org and spacemacs, I want it a top notch integration.

sooheon commented:
After being frustrated by `o` and `O` in evil-org one too many times, I decided to try this. abo-abo is a genius, and it's a shame he doesn't completely embrace vi. Still, this is another paradigm beyond emacs or vi, and it may only be appropriate as an option, not the default (it doesn't use normal mode, but another *special* mode in insert mode). Once you learn it, you love it though.

Devagamster commented:
I think this should be a layer of its own instead of being merged directly into the org layer, that way you can choose either the org bindings as they are now or the worf bindings which take over a lot more stuff. An added benefit is that this could get merged pretty quickly since it is totally optional instead of screwing everyone's bindings which depend on org mode.

sooheon commented:
I agree with making this an additional layer for all the reasons you've said, and that the layer can deal specifically with compatibility issues between the packages.

CestDiego commented:
What do you think about having a `layer-variable` that lets the user select either `evil-org` or `worf` as the keybindings? or actually just chose none?

CestDiego commented:
Ok guys, I added the variable anyway :P @geo7 this way you can "try" evil-org as well :P. Also I added the `,` keybindings for org into the common `init-org` as I think worf might benefit from it also

robbyoconnor commented:
I like this -- I want this

robbyoconnor commented:
:+1:

CestDiego commented:
@TheBB we got approval of the PR police :P

Devagamster commented:
I too want this :+1:

Devagamster commented:
To worf org not to worf. That is the question

TheBB commented on contrib/org/packages.el:
```diff
@@ -22,6 +22,17 @@
     ))
 
+(when org-mapping-style
+  (push org-mapping-style org-packages))
```
A bit too magical maybe? I would rather `worf` and `evil-org` are both in `org-packages`. You can use `:if` in `use-package` to pick which one to install, or just make the defuns conditional on the value of `org-mapping-style`. That way both packages still show up in `SPC f e h`.

CestDiego replied:
I like this approach much better thanks for the feedback :)

CestDiego replied:
#done :)

robbyoconnor replied:
Yay!
On 08/26/2015 04:12 PM, Diego Berrocal wrote:
> In contrib/org/packages.el
> <https://github.com/syl20bnr/spacemacs/pull/1417#discussion_r38028998>:
>
> > @@ -22,6 +22,17 @@
> >      toc-org
> >      ))
> >  
> > +(when org-mapping-style
> > +  (push org-mapping-style org-packages))
>
> #done :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/1417/files#r38028998>.
>

CestDiego commented:
@syl20bnr  now that we have evil-org i think this basically isn't very important?

a13ph commented:
@CestDiego worf is similar, but different. It's more emacs-style, since it is a different way of doing orgmode speedkeys. You do worf things from inside insert state, but while on heading stars. Evil-org is very incomplete from what i've seen, compared to worf. 
There's also `syndicate` and maybe also 1-2 other other implementations...

zestyr commented:
Any update on this?

TheBB commented:
Thanks for this ancient PR. I'll close it now because it has served its purpose. https://github.com/syl20bnr/spacemacs/pull/9041
